src/UI/ledger/LedgerLoginContainer/AddressTable.tsx

```diff
@@ -27,6 +28,8 @@ export interface AddressTablePropsType extends WithClassnameType {
   onGoToPrevPage: () => void;
   onGoToNextPage: () => void;
   onConfirmSelectedAddress: () => void;
+  customContentComponent?: ReactNode;
+  addressTableComponentsClasses?: InnerLedgerComponentsClassesType;
```

Comment:
why `addressTableComponentsClasses` is not called `innerLedgerComponentsClasses` ?

Reply:
Changed. 👍